README.md

```diff
@@ -13,20 +13,33 @@ elsewhere too.
 Having a clear Mozilla position on these specifications helps us align our thinking and communicate
 it clearly to these standards bodies, as well as other browsers.
 
-*See our [contribution guidelines](CONTRIBUTING.md) for information about how you can participate.*
-
 Implementation status (or even intention) isn't tracked here; see [dev-platform](https://groups.google.com/a/mozilla.org/g/dev-platform/).
 
 ### Possible Specification Positions
 
-The currently possible positions are:
+We will seek to apply one of three labels to new specifications:
+
+- `positive` - Mozilla regards this work as a potential improvement to the web.
+- `neutral` - Mozilla is not convinced of the merits of this work, but does not see any significant negative potential.
+- `negative` - Mozilla believes that pursuing this work in its current form would not be good for the web.
+
+These positions will be based on our best assessment of the current state of the specification and
+the potential for it to benefit our users and the web as a whole.  We consider the intended purpose
+and the design fundamentals of each specification.  Incomplete documentation, minor issues, or lack
+interoperability testing are not reasons for a negative position, provided that intent is clear.
+
+We might also use the following labels for specifications where we do not have enough information or
```

Comment:
I like splitting these out into a group separate from the others. But the old definition for `defer` includes a lot of discretion for the reason to not set a position - it's just "does not intend to" (and even stronger with "at all"). This new group header text narrows this quite a bit by basing it on missing information. I can think of cases where any information we'd need to form a position is accessible to us, but we choose not to invest the time to do it. In some sense the text here is still accurate - we don't have enough information, but that's because we aren't seeking it. And I think defined this way it may be construed as a desire for people looking for our position to just send over more information if something is deferred.

WDYT? If we want to keep closer semantics to what we have today we could prefix the group with something simpler like "We may use the following labels for specifications where we haven't formed a position".

Reply:
Yeah, I think that is easier.  I wanted to give more context, but it turns out that trying to enumerate the list failed to be complete.  Sometimes we just don't want to, but putting that into the README doesn't sound good.